test/unit/tasks/seeds_test.rb

```diff
@@ -35,11 +39,11 @@ def seed
     seed
 
     tables.each do |model|
-      refute model.unscoped.count.zero?
+      assert_not model.unscoped.count.zero?
```

Comment:
iiuc, the difference between `refute` and `assert_not` is that refute expects `false` while `assert_not` also accepts `nil`. Is there a condition in which this and other assertions you changed return `nil` instead of `false`?

Reply:
Good point, I haven't found any.